Source/Core/DolphinWX/Debugger/CodeWindowFunctions.cpp

```diff
@@ -135,7 +135,7 @@ void CCodeWindow::OnProfilerMenu(wxCommandEvent& event)
 
     if (Core::GetState() == Core::State::Paused && PowerPC::GetMode() == PowerPC::CoreMode::JIT)
     {
-      if (g_jit != nullptr)
+      if (JitInterface::GetCore() != nullptr)
```

Comment:
It seems to just be old code from [back in 2011](https://github.com/dolphin-emu/dolphin/commit/ba54fac9eb74be6f73e20e2de953cf6f355be57f#diff-7ed92e51c2472de472c9640f33ac86d8R201) that existed before the outer if statement. I'll remove this check.